test/test_helper.exs

```diff
@@ -1,5 +1,5 @@
 ExUnit.start()
-Mox.defmock(MockCollector, for: Station.Collector)
-Mox.defmock(MockRegister, for: Station.Registry)
+Mox.defmock(MockCollector, for: Station.CollectorBehaviour)
```

Comment:
If we put the mocks here, don't they become global?

Reply:
When one refers to the mocks being global or process private it refers to the expectations defined on the mocks.
For Example if the mocks being defined in test_helper.exs made the mocks expectations global then there wouldn't be a need for ```allow/3```, but without allow the tests will fail (an Exception is raised saying that the the mock module  is not available).